src/Bundle/DependencyInjection/SyliusResourceExtension.php

```diff
@@ -141,7 +141,7 @@ private function autoRegisterResources(array &$config, ContainerBuilder $contain
                         'factory' => Factory::class,
                         'form' => DefaultResourceType::class,
                     ],
-                    'driver' => 'doctrine/orm',
+                    'driver' => $resource->getDriver() ?? 'doctrine/orm',
```

Comment:
(I know I'm picky with this one) Can we use constant instead of this raw string ?

Reply:
Done. I don't like the fact that these constants are part of the bundle.